File: DKCSharp/functions/dk_fileExists.cs
public static class APP{
        	
//############################################################################
//# dk_fileExists(file)
//#

public static bool dk_fileExists(string file){
	bool exists = System.IO.File.Exists(file);
	System.Console.WriteLine("{0} exists = {1}", file, exists);
	return exists;
}
    
	
	
	
	
//###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
public static void DKTEST(){
	bool exists = dk_fileExists("C:\\Windows\\System32\\cmd.exe");
	System.Console.WriteLine("exists = "+exists);
}

}

File: DKC/functions/dk_fileExists.c
#include "dk_fileExists.h"


//###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
#ifndef DKMAIN
#define DKMAIN = 1
int main(int argc, char **argv) {
	
	if(dk_fileExists("/C/Windows/System32/cmd.exe")){
		printf("file exists\n");
	} else {
		printf("file does NOT exist\n");
	}
	
	if(dk_fileExists("/C/Non/Existent/File")){
		printf("file exists\n");
	} else {
		printf("file does NOT exist\n");
	}
	
    return 0;
}
#endif
